influxdb/Dockerfile.raspberrypi4-64

```diff
@@ -0,0 +1,7 @@
+FROM balenalib/%%BALENA_MACHINE_NAME%%:buster
```

Comment:
Why not rather hardcode to armhf alpine image used in the pi3? Don't think it's worth using buster just to have aarch64.

Reply:
Yeah, that's a good idea